src/main/java/com/scalar/cassy/server/CassyServerModule.java

```diff
@@ -48,7 +48,6 @@ CassyServerConfig provideCassyServerConfig() {
   }
 
   @Provides
-  @Singleton
   Connection provideConnection() throws SQLException {
```

Comment:
It looks like this `provideConnection` method is no longer used as we create a `Connection` object in `CassyServerController.java` every time we need. Can we remove this method?

Similarly, the annotations `@Inject` on the constructors of `BackupHistory` and `ClusterInfo` and `RestoreHistory`  can be removed as these instances are no longer created by Guice?

Reply:
> It looks like this provideConnection method is no longer used as we create a Connection object in CassyServerController.java every time we need. Can we remove this method?

RemoteCommandHandler still uses Connection injected by Guice, but it might be confusing to create some connections through Guice and other connections manually. What do you think?
 
> Similarly, the annotations @Inject on the constructors of BackupHistory and ClusterInfo and RestoreHistory can be removed as these instances are no longer created by Guice?

Good point! I'll remove @Inject from those. Thank you!